src/glossary.js

```diff
@@ -145,6 +145,7 @@ Glossary.prototype.handleTermTouch = function(e) {
   if (e.which === KEYCODE_ENTER || e.type === 'click') {
     if (selectorMatches(e.target, '[data-term]')) {
       this.show();
+      this.$selectedTerm = $(e.target);
```

Comment:
Oh I realized when tests were failing so it's already updated without $ objects.